p2p/Cargo.toml

```diff
@@ -19,13 +19,11 @@ description = """
     The Tendermint P2P stack in Rust.
     """
 
-[lib]
-test = false
```

Comment:
Not sure why this was here, but I added a test so I had to remove this to make the test work

Reply:
The reason it is declared is that all tests are in a separate crate over at https://github.com/informalsystems/tendermint-rs/tree/master/test

Reply:
Aah, ok.

My test was mostly useful for ensuring message equivalence during development. Since I'm fairly confident of that at this point (and hopefully all of this can be removed soon) I can remove it and restore these lines.

Reply:
Sounds good!

Reply:
Or keep the test and add it to the `test` crate?

Reply:
@thanethomson the API the test is written against isn't public.

I went ahead and removed it for now.